### DISCUSSION TITLE
End request sent only to the winner snake

### DISCUSSION THREAD

**Discussion by Nettogrof** [BattlesnakeOfficial/feedback#32] [Bug Reports]
Currently the '/end' request is send only is the "winner snake"

`for _, snake := range state.Snakes {
			if snake.EliminatedCause == rules.NotEliminated {
				isDraw = false
				winner = Battlesnakes[snake.ID].Name
				sendEndRequest(state, Battlesnakes[snake.ID])
			}
		}`

According to the doc: 
**Your Battlesnake will receive this request whenever a game it was playing has ended. Use it to learn how your Battlesnake won or lost and deallocated any server-side resources. Your response to this request will be ignored.**

**Comment by bvanvugt** [comment]
👍 we should fix this!

**Comment by bvanvugt** [comment]
Should be fixed in [Rules v1.0.18](https://github.com/BattlesnakeOfficial/rules/releases/tag/v1.0.18)
